velox/vector/arrow/Bridge.cpp

```diff
@@ -972,7 +972,17 @@ TypePtr importFromArrowImpl(
         // Parse "d:".
         int precision = std::stoi(&format[2], &sz);
         // Parse ",".
-        int scale = std::stoi(&format[2 + sz + 1], &sz);
+        int idx = 2 + sz + 1;
+        int scale = std::stoi(&format[idx], &sz);
+        // Handle bitwidth.
```

Comment:
nit: bitwidth is not handled, as we only check if it is equal to the expected value. Maybe change to sth. like `If bitwidth is provided, check if it is equal to 128.`